app/controllers/event_controller.rb

```diff
@@ -0,0 +1,52 @@
+class EventController < ApplicationController
+  before_action :session?
+  def index
+    @new_list = Event.where('events.event_date >= ?', DateTime.current.beginning_of_day)
+    @old_list = Event.where('events.event_date < ?', DateTime.current.beginning_of_day)
+    @users = User.all
+    @attendee = Attendee.where(user_id: cookies[:userid])
+  end
+
+  def new
+    @event = Event.new
+  end
+
+  def create
+    @event = Event.new(user_params)
```

Comment:
You should use the `build` method instead of Event.new. That way, you can associate the event to its creator in one line. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/build_method_requirement.png) from Odin)